Pipfile

```diff
@@ -8,10 +8,11 @@ name = "pypi"
 [packages]
 
 Flask = "*"
-jsonschema = "*"
+jsonschema = "<3"
 structlog = "*"
 python-dateutil = "*"
-
+jsonpointer = "*"
+eq-translations = {editable = true,git = "https://github.com/ONSDigital/eq-translations.git",ref = "master"}
```

Comment:
Does eq-translations need to be installed with the editable flag? This has the effect of adding all the eq-translation dependencies to the Pipfile.lock

Reply:
Removed this